zokrates_core_test/tests/tests/split_bls.json

```diff
@@ -1,5 +1,5 @@
 {
-	"entry_point": "./tests/tests/split_bls.zok",
+	"entry_point": "./tests/tests/split.zok",
 	"curves": ["Bls12_381"],
```

Comment:
Can this now be run on more curves?

Reply:
There are two `json` test files (`split_bls.json` and `split_bn.json`) that use the same `zok` file (`split.zok`). If we were to specify both curves in the `curves` array we would have failing tests, because return values would not match. The way tests are structured now does not allow us to specify a curve for a specific test, only to specify many tests that would all run on each curve.

Reply:
right!